src/constraint.jl

```diff
@@ -26,6 +26,11 @@ Base.@kwdef mutable struct Constraint{V}
     end
 end
 
+Constraint(v::T, b::Int) where {T<:Value} = Constraint{T}(v, float(b))
+Constraint(v::T, b::Tuple{Int, Int}) where {T<:Value} = Constraint{T}(v, float.(b))
+Constraint(v::T, b::Tuple{Int, Float64}) where {T<:Value} = Constraint{T}(v, float.(b))
+Constraint(v::T, b::Tuple{Float64, Int}) where {T<:Value} = Constraint{T}(v, float.(b))
```

Comment:
```suggestion
Constraint(v::T, b::Real) where {T<:Value} = Constraint{T}(v, Float64(b))
Constraint(v::T, b::Tuple{A,B}) where {A<:Real,B<:Real} = Constraint{T}(v, Float64.(b))
```

- use exact target datatype
- we've got `Real` :]

Reply:
btw this still needs tests, pls just change some of the floats in the docs to ints so that this triggers.

Reply:
fixed!